src/server.h

```diff
@@ -57,6 +57,13 @@
 #include <systemd/sd-daemon.h>
 #endif
 
+/* Server name */
+#ifdef SERVER_DEF_NAME
+#define SERVER_NAME STRINGIFY(SERVER_DEF_NAME)
+#else
+#define SERVER_NAME "redis"
```

Comment:
do we still need a default here? Makefile has already covered the default case. this seems redundant.

Reply:
Does anyone uses custom Makefile instead of the default one? For such scenario this code would be helpful.

Reply:
redis is fine and preferred IMO. This is just the baby step and in no way a long term commitment. we should get the ball rolling

Reply:
Another option would be to change the second #define to #error "Must provide a SERVER_DEF_NAME value" - this would allow the "redis" string to be removed here with clear directions to resolving the issue for anyone in that fallback case.